packages/itwinui-css/src/slider/slider.scss

```diff
@@ -102,37 +110,39 @@ $tick-height: $iui-baseline; // 12px
     cursor: not-allowed;
     background-color: var(--iui-color-background-4);
     border-color: var(--iui-color-background-4);
+    @media (forced-colors: active) {
```

Comment:
This might have been a bad merge in the v1 branch from when the vertical slider PR was merged into main
https://github.com/iTwin/iTwinUI/blob/3b9b327ed8cf84db87dc1d61d356445cd5b051ba/packages/itwinui-css/src/slider/slider.scss#L121